pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "cmake==3.18.0", "ninja"]
```

Comment:
There isn't a pip package for make that I'm aware of, while KitWare ensures there is one available for ninja (for scikit-build). This promises that a build system generator will always be available. We don't actually every _use_ it, since we directly install from CMake, but it needs to be there for CMake to configure.

Reply:
Right, OK, I see. Minor disadvantage is that it will always install it, even if `make` is installed. But yeah, OK, why not.

Reply:
The PEP 518 requirements should not assume anything about the user's environment. Would make even be that likely to preexist on windows? Certainly might be missing in a new install or docker container.